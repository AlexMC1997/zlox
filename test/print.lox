print "hello there" == "hello sir" or "woot" == "woot" and "Alex M." == "Alex" + " " + "M.";
print "\n";
print "what is up" + " " + "my" + " friend" + "\n";
print 3 * 3 - 4 + (48.2 / 3);
print "\n";
print true and true and false;
print "\n";