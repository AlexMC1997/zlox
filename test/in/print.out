true
what is up my friend
2.1066666666666666e+01
false
